### DISCUSSION TITLE
Welcome to ClimaAtmos.jl Discussions!

### DISCUSSION THREAD

**Discussion by bischtob** [CliMA/ClimaAtmos.jl#109] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by waywardpidgeon** [comment]
I had a lot of experience in coding (mostly Lisp and Fortran) in my younger years. Now I want to work on climate change in Julia. I have explored Oceananigans.jl and now want to look at ClimaAtmos.jl. 
However I am failing to install it under win 10. All goes well (git clone etc) until the last step when it give an incompatibility with ClimaComms error when trying to install ClimaAtmos, v0.3.5(<v0.4.0). After removing ClimaComms and trying to load ClimaAtmos I get "ClimaAtmos has the same name or UUID as the active project". 
Thus I have not been able to install the ClimaAtmos package so can't run the tests or other examples. Please help! I have Julia 1.8.5 installed. Many thanks waywardpidgeon/kab@waikato.ac.nz
